metadata/0x0CED6166873038Ac0cc688e7E6d19E2cBE251Bf0.json

```
@@ -30,6 +30,7 @@
     "slug": "bling-series-1",
     "description": "1754 Factory LLC is a Delaware Series LLC created by Davoa Capital, which is founded by Fabien Dureuil. 1754 Factory will launch Bling Series 1 LLC and enter into an institutional purchase agreement to buy Bling bonds. Bling is a mobile app for French consumers that offers a cash advance system up to €100. Bling bonds are sold at par at €10,000 and they include over 125 micro loans that average €80. Bling Series 1 will launch with 30 Bling bonds and intends to steadily grow the pool.",
     "logo": "https://raw.githubusercontent.com/centrifuge/tinlake-pools-mainnet/main/assets/1754-factory.png",
+    "icon": "https://storage.googleapis.com/tinlake/pool-icons/1745.svg",
```

We'll now have 4 images (at least) per pool: logo for on pool overview, icon for dashboard, TIN and DROP token icons. I think we should migrate all these to GCS (so also the logos which are currently in Github), and I think it would be best to start using something like this format:

```suggestion
    "media": {
        "logo": "https://storage.googleapis.com/tinlake/pool-icons/1745/logo.svg",
        "icon": "https://storage.googleapis.com/tinlake/pool-icons/1745/icon.svg",
        "dropToken": "https://storage.googleapis.com/tinlake/pool-icons/1745/drop.svg",
        "tinToken": "https://storage.googleapis.com/tinlake/pool-icons/1745/tin.svg"
      }
```

---

Great idea, changed below